core/templates/pages/about-foundation-page/about-foundation-page.component.html

```diff
@@ -282,7 +286,9 @@ <h1>
     text-decoration: underline;
   }
 
-
+  figcaption {
+    text-align: center;
+  }
```

Comment:
can you please use class instead of `figcaption`

Reply:
Ok ,I have made the changes @heyimShivam @SanjaySajuJacob and I have attached the video of it.
[Screencast from 2023-03-25 20-37-08.webm](https://user-images.githubusercontent.com/67548071/227727046-6e4f7910-1fa7-4d73-87d4-cde9d7185414.webm)